csrc/torch_bindings.cpp

```diff
@@ -9,9 +9,10 @@
 
 namespace kvcached {
 
-void init_kvcached(const std::string &dev_str, size_t page_size = 0) {
+void init_kvcached(const std::string &dev_str, size_t page_size = 0,
+                   bool contiguous_layout = false) {
```

Comment:
Here, the contiguous_layout flag is false by default, but in the python interfaces, when shutdown kvcached, the default value is set to be true.

Reply:
Good catch! I have modified it to `true` to keep aligned.